src/fungi/meta.rs

```diff
@@ -348,7 +348,10 @@ impl Writer {
             .journal_mode(SqliteJournalMode::Delete)
             .filename(path);
 
-        let pool = SqlitePool::connect_with(opts).await?;
+        let pool = SqlitePoolOptions::new()
+            .max_connections(1)
```

Comment:
since there is already an explicit `opts` variable it's slightly confusing to see this option added here instead of adding it to `opts`.